USAGE.md

```diff
@@ -21,24 +27,33 @@ The full list of steps from initial file upload to celebrating the success can b
 8.  Check that product appeared in the connected system.
 9.  Celebrate!
 
-### Offers
+#### Offers
 1.  Place your file into the source folder
 2.  Run the tool 
 
         ./product-catalog-import-tool
 
 2.  If the file is correct then offers will be created
 
-### Products and Offers
+#### Offer Items
+1.  Place your file into the source folder
+2.  Run the tool 
+
+        ./product-catalog-import-tool
+
+2.  If the file is correct and offers and products exist alrteady in the system then offer items will be created
+
+### Offers, Products and OfferItems
 1. You need to differentiate source folders in the configuration file:
 
         product:
-            source: ./data/source/products
+            source: ./data/source/products/
         offer:
-            source: ./data/source/offers
+            source: ./data/source/offers/
+        offer_item:
+          source: ./data/source/offerItems/
 
-3. Repeat steps from Offers part
-4. Repeat steps from Products part
+3. Place all files and run the tool just once. Files will be imported in the following order: offers, products, offer items.
```

Comment:
_4.  Result - this folder contains results of validation and transformation with the following subfolders:
    *  ‘report’ - while a file has any missing attributes then the tool will keep it in this folder with all recommendations to fix; 
._
    
For product import-we make report in case of failed import also